README.md

```diff
@@ -2,5 +2,50 @@
 
 [![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
 
-This repository will contain the source code of GitHub Apps designed
+This repository contains the source code of GitHub Apps designed
 specifically for the AMP Project.
+
+## Apps
+
+### [Bundle-size Bot](bundle-size/README.md)
+
+Contact: [@danielrozenberg](https://github.com/danielrozenberg)
+
+This GitHub App performs checks on the size of the compiled AMP bundle, and blocks pull requests from being merged without specific approval from the team that maintains the bundle size.
+
+The app runs on an instance of Google AppEngine and is installed exclusively on the `ampproject/amphtml` repository.
+
+### [Owners Bot](owners/README.md)
+
+Contact: [@rcebulko](https://github.com/rcebulko)
+
+A web service that suggests approvers for a GitHub pull request based on OWNERS files and enforces reviews by OWNERS as a GitHub status check.
+
+The app runs on an instance of [Google AppEngine](https://ampproject-owners-bot.appspot.com) and is installed exclusively on the `[ampproject/amphtml](https://github.com/ampproject/amphtml)` repository. It can be trivially deployed to other organizations and repositories.
+
+### [PR Deploy Bot](pr-deploy/README.md)
+
+Contact: [@estherkim](https://github.com/estherkim)
+
+A GitHub App which deploys a PR branch to a static website when you ask it to. 
+
+Websites are served by a [Google Cloud Storage bucket](https://console.cloud.google.com/storage/browser/amp-test-website-1).
```

Comment:
This URL is not publicly accessible

Reply:
I pulled this directly from the PR Deploy Bot README, so I guess that should be updated too. I'm going to remove both hyperlinks in this PR unless @estherkim has a suggestion for something to replace it with.

Reply:
Removing it is fine with me. Thanks.